AddOns/BasicFinancials/app/AppSourceCop.json

```diff
@@ -0,0 +1,4 @@
+{
+    "mandatoryAffixes": ["BF"],
```

Comment:
Test Automation is missing. We need to have at least 80% code coverage

Reply:
Test Automation is not part of the scoop, and has not been accounted for in the project, but is something that should be done. Fredrik Hietala (Microsoft) knows about this.